pages/slider.html

```diff
@@ -0,0 +1,68 @@
+    <link rel="icon" href="https://img.icons8.com/ios-glyphs/50/000000/fast-track.png" />
+    <link rel="stylesheet" href="../styles/main.css">
+    <title>Slider | FAST UI</title>
+</head>
+
+<body>
+    <header>
+        <nav class="navbar">
+            <div class="left-nav">
+                <a href="../index.html" class="nav nav-logo">FAST UI</a>
+                <span class="small-text">V 1.0</span>
+            </div>
+            <div class="right-nav">
+                <ul class="list">
+                    <li><a href="../index.html" class="nav nav-pills">Home</a></li>
+                    <li><a href="./avatar.html" class="nav right-nav-active nav-pills">Documentation</a></li>
+                    <li><a href="./installation.html" class="nav nav-pills">Installation</a></li>
+                </ul>
+            </div>
+        </nav>
+    </header>
+    <div class="sidebar">
+        <div class="items">
+            <p class="subtext">Components</p>
+            <a href="./avatar.html" class="item">Avatar</a>
+            <a href="./alert.html" class="item">Alert</a>
+            <a href="./badge.html" class="item">Badges</a>
+            <a href="./button.html" class="item">Button</a>
+            <a href="./card.html" class="item">Card</a>
+            <a href="./dialogue.html" class="item">Dialogue</a>
+            <a href="./image.html" class="item">Image</a>
+            <a href="./input.html" class="item">Input</a>
+            <a href="./navbar.html" class="item">Navbar</a>
+            <a href="./rating.html" class="item">Ratings</a>
+            <a href="./slider.html" class="item">Slider</a>
+            <a href="./toast.html" class="item">Snackbar</a>
+            <a href="./typography.html" class="item">Typography</a>
+        </div>
+    </div>
+    <div class="main-container-components">
+        <h2 class="heading">Slider</h2>
+        <p class="description">Slider is used for a range of value to present your cost and content as clearly and
+            efficiently as possible.
+        </p>
+        <h3 class="heading heading-avatar">Simple Slider</h3>
+        <p class="description description-avatar">You can use <span class="colored-text"> slider </span> class for
+            headings as shown below. </p>
+        <iframe class="code-snippet-display"
+            src="https://carbon.now.sh/embed?bg=rgba%28255%2C59%2C48%2C1%29&t=base16-light&wt=none&l=htmlmixed&width=823&ds=false&dsyoff=82px&dsblur=100px&wc=true&wa=false&pv=27px&ph=22px&ln=false&fl=1&fm=IBM+Plex+Mono&fs=14px&lh=143%25&si=false&es=2x&wm=false&code=%252F%252F%2520Slider%250A%253Cinput%2520type%253D%2522range%2522%2520min%253D%25221%2522%2520max%253D%2522100%2522%2520value%253D%252250%2522%2520class%253D%2522slider%2522%2520id%253D%2522myRange%2522%2520%252F%253E%250A%2520%2520%2520%2520%2520"
+            sandbox="allow-scripts allow-same-origin">
+        </iframe>
+        <div class="badge_container-main flex-center">
```

Comment:
change the name from badge container to something relevant.

Reply:
This is the class used for particular styling on the section